### DISCUSSION TITLE
Where are the password stored?

### DISCUSSION THREAD

**Discussion by FloatingLizard** [zeapo/Android-Password-Store#1502] [Q&A] [answered]
I am looking for some passwords on my Android device and where are they stored?  

regards,

**Comment by Skrilltrax** [comment]
Hey, I'm guessing you chose Hidden/Internal Storage during the onboarding. If that's the case, you can export your passwords by going to Settings > Export Passwords (at the end of the preferences)

**Reply by FloatingLizard** [reply]
Actually I really need to know where this location is.  I have used this apps in the past and I need to check if there is anything in it.

**Comment by msfjarvis** [comment, marked as answer]
`/data/data/dev.msfjarvis.aps/files/store`

**Reply by FloatingLizard** [reply]
Cool...  Is this accessible or do I need to be rooted or else?  This is a Tab s7

**Reply by msfjarvis** [reply]
It's called 'Hidden' because it is inaccessible. You can browse it with root, export it how Skrilltrax said above, and simply uninstall the app to clear it.

**Reply by FloatingLizard** [reply]
Yeah, I realize that.  I think if I install Studio I should be able to access it.

**Reply by msfjarvis** [reply]
You won't. Studio can only browse app data for debuggable apps, which release builds from Google Play or F-Droid aren't.

**Reply by FloatingLizard** [reply]
adb shell?

**Reply by msfjarvis** [reply]
"Hidden" unequivocally means what it says. The options I gave earlier are exhaustive.

**Reply by FloatingLizard** [reply]
OK thanks..
